.gitlab-ci.yml

```diff
@@ -247,3 +247,4 @@ include:
   - local: '/.gitlab/ci/ubuntu2004.yml'
   - local: '/.gitlab/ci/ubuntu2204.yml'
   - local: '/.gitlab/ci/windows10.yml'
+  - local: '/.gitlab/ci/sync-spack.yml'
```

Comment:
Why do we have a `.gitlab` directory? The repo is in github now.

Reply:
Spack CI runs in a gitlab instance

Reply:
Got it. So a PR's commit is pushed to the spack GitLab instance and the CI is run from there, correct? And through magic that all gets reported back to the GitHub PR.